sample-notificator-domain/src/main/groovy/life/qbic/business/notification/create/NotificationContent.groovy

```diff
@@ -43,7 +43,7 @@ class NotificationContent {
             this.customerEmailAddress = Objects.requireNonNull(customerEmailAddress, "Email address of customer must not be null")
 
             /*Project Information*/
-            this.projectTitle = Objects.requireNonNull(projectTitle, "Project title must not be null")
+            this.projectTitle = projectTitle ?: ""
```

Comment:
Do you think it would make sense to add "not project title defined" or something like this? Would it be more confusing to you (as a receiver of the email) to see an empty field or to see that the value was not set?

Reply:
I think I vote for "No title"

Reply:
https://github.com/qbicsoftware/sample-notificator-cli/pull/36/files#diff-e1782e0961f49bfed8f6d357ce69da0973bbb1e6b36c717102f448d429d0f09bR45 <- we need to be careful with assumptions

Reply:
I dont understand what you mean

Reply:
In the line I mentioned it is checked whether the content of getProjectTitle is empty (as groovy does regard empty strings and nulls as `false`). When you set it to anything but empty the text will change. So this is the wrong place/ the decision is made somewhere else right now

Reply:
I added the suggestion of "No title" to the emailHTMLTemplate class.